utils/calldata/boost_contract_claim.ts

```diff
@@ -0,0 +1,28 @@
+import { CallDetails, cairo, shortString } from "starknet";
+import { decimalToHex } from "../feltService";
+
+function boostContractClaimData(
+  contractAddress: string,
+  boostId: number,
+  amount: number,
+  token: string,
+  signatures: string[]
+): CallDetails {
```

Comment:
You should return a `Call` instead of a `CallDetails` because the starknet-react hook expects a `Call[]`